src/perf/lib/ThroughputServer.cpp

```diff
@@ -47,25 +45,16 @@ ThroughputServer::Init(
     _In_ int argc,
     _In_reads_(argc) _Null_terminated_ char* argv[]
     ) {
-    if (!Listener.IsValid()) {
-        return Listener.GetInitStatus();
-    }
-
-    uint16_t port = THROUGHPUT_DEFAULT_PORT;
-    TryGetValue(argc, argv, "port", &port);
-
-    const char* localAddress = nullptr;
-    if (!TryGetValue(argc, argv, "listen", &localAddress)) {
-        WriteOutput("Server mode must have -listen\n");
+    if (argc > 0 && (IsArg(argv[1], "?") || IsArg(argv[1], "help"))) {
```

Comment:
This check would need to be `argc > 1` to access index 1. But I think the access should be index 0, see my comment below.

Reply:
Ah, yes. It should be `argv[0]`.